OSDOCS-16037 added metrics and log level assemblies and modules (#98474)
openshift/openshift-docs: security/external_secrets_operator/external-secrets-monitoring.adoc

:_mod-docs-content-type: ASSEMBLY
[id="external-secrets-monitoring"]
= Monitoring the External Secrets Operator for Red Hat OpenShift
include::_attributes/common-attributes.adoc[]
:context: external-secrets-monitoring

toc::[]

> By default, the {external-secrets-operator} exposes metrics for the Operator and the operand. You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.

Reviewer: nit: should we list the operands? Currently we have just one, but in future we may add couple more.

```suggestion
By default, the {external-secrets-operator} exposes metrics for the Operator and the operands(external-secrets). You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.
```

> By default, the {external-secrets-operator} exposes metrics for the Operator and the operands ('external-secrets'). You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.

Author: Deleted what was in parenthesis.

// Metrics scraping for external-secrets-operator
include::modules/external-secrets-enable-operator-metrics.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

* xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]

// Querying metrics for the external-secrets operator
include::modules/external-secrets-query-operator-metrics.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

* xref:../../observability/monitoring/accessing-metrics/accessing-metrics-as-an-administrator.adoc#accessing-metrics[Accessing metrics]

> // Enabling user workload monitoring for the external-secrets-operator operand

Reviewer: This should be the first sub-section under `Monitoring the External Secrets Operator for Red Hat OpenShift` section.

> * xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]

Reviewer 2: This title doesn't match up to the page's title

include::modules/external-secrets-enable-user-workload-monitor.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources
* xref:../../observability/monitoring/configuring-user-workload-monitoring/configuring-metrics-uwm.adoc#setting-up-metrics-collection-for-user-defined-projects_configuring-metrics-uwm[Setting up metrics collection for user-defined projects]

> // Metrics scraping for cert-manager operands by using a ServiceMonitor

Reviewer: ```suggestion
// Metrics scraping for external-secrets operand by using a ServiceMonitor
```

Author: Done

include::modules/external-secrets-enable-metrics.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

* xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]

> // Querying metrics for the cert-manager operands

Reviewer: ```suggestion
// Querying metrics for the external-secrets operand
```

Author: Done
